openstf/pipeline/predict_sklearn.py

```diff
@@ -1,18 +1,19 @@
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from pathlib import Path
 
 import joblib
 import pandas as pd
-
+import structlog
 
 from ktpbase.database import DataBase
-from openstf.pipeline.create_forecast import generate_forecast_datetime_range, generate_inputdata_datetime_range
-from openstf.validation.validation import validate, clean, is_data_sufficient
+from openstf.validation.validation import validate, is_data_sufficient, find_nonzero_flatliner
```

Comment:
I think its more clear if you import the validation module instead of the individual functions. By doing so, its more clear where these function came from while you use them in the pipeline:

```python
from openstf.validation import validation

# in the code
validation.validate()
validation.is_data_sufficient()
# and so on
```